shared/fs/folder/offline.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as Kb from '../../common-adapters'
+import React from 'react'
+import * as Styles from '../../styles/index'
+
+// TODO: what is the correct thing to do when don't have any props
+class OfflineFolder extends React.PureComponent<any> {
+  render() {
+    return (
+      <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true}>
+        <Kb.Box2
+          direction="vertical"
+          style={styles.emptyContainer}
+          fullHeight={true}
+          fullWidth={true}
+          centerChildren={true}
+        >
+          <Kb.Icon type="iconfont-cloud" fontSize={64} color={Styles.globalColors.black_10} />
```

Comment:
I'm guessing this needs to be `48` on mobile. But really we should probably have `Huge` as `sizeType` for the `Kb.Icon`. cc @adamjspooner thoughts?

Reply:
I found where to add 'Huge' as an option, but I can't find where the definitions mapping names to numbers are. Where can I find that?

Reply:
Oh, saw `typeToFontSize` on another PR.